src/kernels/kernel.ts

```diff
@@ -191,7 +191,6 @@ abstract class BaseKernel implements IBaseKernel {
             Array.from(this.hooks.get('willInterrupt') || new Set<Hook>()).map((h) => h())
         );
         traceInfo(`Interrupt requested ${getDisplayPath(this.resourceUri || this.uri)}`);
-        this.startCancellation.cancel();
```

Comment:
We don't need to cancel when interrupting a kernel, its wrong.
If we do i'd like to see more information (comments/tests for that case).
I'm removing this here as its not requried.